javalanglib/src/main/scala/java/lang/Long.scala

```diff
@@ -111,8 +111,7 @@ object Long {
   }
 
   // Intrinsic
-  @inline def toString(i: scala.Long): String =
-    toStringImpl(i, 10)
+  @inline def toString(i: scala.Long): String = "" + i
```

Comment:
Because `toStringImpl` (transitively) depends on a bunch of things: `JSNumberOps`, `JSStringOps`, `StringRadixInfo` which is a `js.Array` (and whose rhs depends on `Range` and its `foreach`.

Note that with this change, `Long.toString(i)` has now an implementation similar to `Integer.toString(i)`, which is nice.